runbotics/runbotics-desktop/src/action/microsoft/automation/excel/cloud-excel.action-handler.ts

```diff
@@ -60,7 +60,7 @@ export class CloudExcelActionHandler extends StatefulActionHandler {
     }
 
     setCell(input: CloudExcelTypes.CloudExcelSetCellActionInput) {
-        return this.excelService.setCell(this.session, input.cell, input.content);
+        return this.excelService.setCell(this.session, input.cell, input.value);
```

Comment:
`this.excelService.setCell(this.session, input.cell, input.value)` returns `void`, imo we don't need the `return` here